mycore-iview2/src/main/java/org/mycore/iview2/frontend/configuration/MCRViewerBaseConfiguration.java

```diff
@@ -55,6 +55,11 @@ public MCRViewerConfiguration setup(HttpServletRequest request) {
             setProperty("canvas.startup.fitWidth", true);
         }
 
+        String leftShowOnStart = MCRConfiguration.instance().getString("MCR.Viewer.leftShowOnStart", null);
+        if(leftShowOnStart!=null){
```

Comment:
setProperty() should handle null values to eliminate all "if (value != null)" checks